tests/test_server.py

```diff
@@ -1,14 +1,18 @@
 import os
+import sys
 from pathlib import Path
 
 import pytest
+from pytest import MonkeyPatch
 from flaky import flaky
 
 from teachbooks.serve import Server
 
 SERVE_DIR = Path(".")
 WORK_DIR = Path("./.teachbooks")
 
+sys.argv = ['serve.py', './']
```

Comment:
is your assumption that we would be able to pass the path as an argument when calling serve.py from a CLI?

Reply:
Yes, this is the test scenario for sys.argv